pact/concrete-policies/collection-policy/collection-policy-v1.repl

```diff
@@ -13,6 +13,13 @@
   (marmalade-v2.abc.create-account "k:creator" (read-keyset 'creator-guard))
 (commit-tx)
 
+(begin-tx)
```

Comment:
These concrete policies should have access to `mini-guard-utils`, because it was loaded in marmalade.repl. so instead of deploying g-utils, we can `(use mini-guard-utils)`